### ISSUE TITLE
Ability to include snippets from separate files

### ISSUE THREAD

**Issue by danielbayley** [atom/snippets#215]
I'm not talking about including other .cson snippets files (my _[modular-snippets]_ package already does that…), but simply plain text from _any_ file type as the actual snippet _content_. For example:
~~~ coffee
'.source.coffee':
  snippet:
    prefix: 'prefix'
    body: 'include.coffee' # ./relative path to included file
~~~

[modular-snippets]: https://atom.io/packages/modular-snippets

**Comment by mnquintana** [comment]
Thanks for the suggestion! This looks like a duplicate of https://github.com/atom/snippets/issues/124 – feel free to subscribe there for updates.

**Comment by danielbayley** [comment]
@mnquintana This _isn't_ a duplicate… It's a different feature. Plus I actually already implemented #124 in my own package, as mentioned…
